roles/docker/tasks/install-docker-Debian.yml

```diff
@@ -180,3 +180,9 @@
   when:
     - docker_registry_username is defined and docker_registry_username|string|length > 0
     - docker_registry_password is defined and docker_registry_password|string|length > 0
+
+- name: Ensure that some packages are not installed
+  become: true
+  ansible.builtin.apt:
+    name: ["docker-ce-rootless-extras", "docker-buildx-plugin"]
```

Comment:
nits: wrong order (b<c), also make an explicit list with one entry per line to easy future changes?